enums/enumgen/methods.go

```diff
@@ -45,8 +45,10 @@ func (i {{.Name}}) {{if .IsBitFlag}} BitIndexString {{else}} String {{end}} () s
 `))
 
 var NConstantTmpl = template.Must(template.New("StringNConstant").Parse(
-	`//{{.Name}}N is the highest valid value for type {{.Name}}, plus one.
+	`//gosl:start
+//{{.Name}}N is the highest valid value for type {{.Name}}, plus one.
 const {{.Name}}N {{.Name}} = {{.MaxValueP1}}
+//gosl:end
```

Comment:
You won't get away with indelibly adding gosl bloat to enumgen.

Reply:
it is literally just comments. 2 of them. in a generated file. get over it. It is _essential_ because you often need these N values in index computations into enum-based tensor indexing.